portal-api/package-lock.json

```diff
@@ -18,6 +21,8 @@
         "dotenv": "16.0.0",
         "express": "4.17.2",
         "filesize": "^8.0.7",
+        "graphql": "^15.8.0",
+        "graphql-middleware": "^4.0.2",
```

Comment:
Is this change needed? Thought I removed graphql/apollo from portal api

Reply:
currently in main->portal-api->package.json .  I did a npm install